Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj

```diff
@@ -280,7 +281,7 @@
     <Compile Include="Startup\AzureCloudOdsStartup.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Common\EdFi.Common.csproj">
+      <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Common\EdFi.Common.csproj">
```

Comment:
remove xtra white space and fix formatting

Reply:
Fixed it .Thanks